cmake/Hunter/config.cmake

```diff
@@ -32,7 +32,8 @@ hunter_config(
 
 hunter_config(
     libp2p
-    VERSION 0.1.6
+    URL https://github.com/libp2p/cpp-libp2p/archive/48f3457dd79fe8b06cf004b7188bfe1e2e9abf48.zip
```

Comment:
????
Why not to introduce a new version in soramitsu-hunter as it was done before?

Reply:
Fixed